tests/snowpark/test_common.py

```diff
@@ -100,15 +103,15 @@ def test_check_if_replace_is_required(mock_procedure_description, arguments, exp
         "snowflake_dependencies": ["snowflake-snowpark-python", "pytest<9.0.0,>=7.0.0"],
         "external_access_integrations": [],
         "imports": [],
-        "stage_artifact_file": "@FOO.BAR.BAZ/my_snowpark_project/app.zip",
+        "stage_artifact_files": ["@FOO.BAR.BAZ/my_snowpark_project/app.zip"],
         "runtime_ver": "3.10",
         "execute_as_caller": True,
     }
     replace_arguments.update(arguments)
 
     assert (
         check_if_replace_is_required(
-            ObjectType.PROCEDURE, mock_procedure_description, **replace_arguments
+            "procedure", mock_procedure_description, **replace_arguments
```

Comment:
Why not `ObjectType.PROCEDURE`?

Reply:
I want to get rid of it because it duplicates information from function and procedure entities. Will do so in followup Pr